ush/forecast_predet.sh

```diff
@@ -556,10 +556,21 @@ FV3_predet() {
         cpreq "${PARMgfs}/ufs/post_itag_gfs" "${DATA}/itag" # TODO: Need a GEFS version when available in the UFS-weather-model
         # TODO: These should be replaced with ones from the ufs-weather-model when available there
         case ${NET} in
-            gfs | gcafs)
+            gfs)
                 cpreq "${PARMgfs}/post/gfs/postxconfig-NT-gfs-two.txt" "${DATA}/postxconfig-NT.txt"
                 cpreq "${PARMgfs}/post/gfs/postxconfig-NT-gfs-f00-two.txt" "${DATA}/postxconfig-NT_FH00.txt"
-                ;;
+              ;;
+            gcafs)
+                cpreq "${PARMgfs}/post/gcafs/postxconfig-NT-gcafs.txt" "${DATA}/postxconfig-NT.txt"
+                cpreq "${PARMgfs}/post/gcafs/postxconfig-NT-gcafs.txt" "${DATA}/postxconfig-NT_FH00.txt"
+                cpreq "${PARMgfs}/ufs/post_itag_gcafs" "${DATA}/itag"  # TODO: Need a GCAFS version when available in the UFS-weather-model
```

Comment:
```suggestion
                cpreq "${PARMgfs}/ufs/post_itag_gcafs" "${DATA}/itag"
```
TODO is not necessary as the file is being linked from the ufs-weather-model in `link_workflow.sh`

Reply:
@aerorahul  We used the same way as the GFS model for "post_itag_gfs" , which is also has been linked in link_workflow.sh. I think we need both of them, the link_workflow.sh is linking the post_itag_gcafs from ufs-weather-model to ${PARMgfs}/ufs/ , while here the "cpreq "${PARMgfs}/ufs/post_itag_gcafs" "${DATA}/" is copy the post_itag_gcafs to RUNDIR as itag. So we can not remove the #TODO